instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Post from './Post'
+import './PostContainer.css'
+
+
+function PostContainer(props) {
+      return (
+        <>
+          <div className="post-container">
+            <Post data={props.data} />
+          </div>
+
+
+        </>
+      );
```

Comment:
I was getting errors from the fragments. They are not needed if you only have one div.

Reply:
OK will keep that in mind.  Fragments removed. Comment addressed.